package.json

```diff
@@ -3,54 +3,25 @@
   "version": "1.0.0",
   "author": "Optimism PBC",
   "license": "MIT",
-  "workspaces": {
-    "packages": [
-      "packages/*",
-      "batch-submitter",
-      "bss-core",
-      "gas-oracle",
-      "integration-tests",
-      "l2geth-exporter",
-      "l2geth",
-      "ops/docker/rpc-proxy",
-      "ops/docker/hardhat",
-      "ops/docker/go-builder",
-      "ops/docker/js-builder",
-      "ops/docker/ci-builder",
-      "ops/docker/foundry",
-      "endpoint-monitor"
-    ],
-    "nohoist": [
-      "**/typechain/*",
-      "**/@typechain/*",
-      "@eth-optimism/contracts-bedrock/ds-test",
-      "@eth-optimism/contracts-bedrock/forge-std",
-      "@eth-optimism/contracts-bedrock/@rari-capital/solmate",
-      "**/@openzeppelin/*",
-      "@eth-optimism/contracts-periphery/ds-test",
-      "@eth-optimism/contracts-periphery/forge-std",
-      "@eth-optimism/contracts-periphery/@rari-capital/solmate",
-      "forta-agent"
-    ]
-  },
   "private": true,
+  "engines": {
+    "pnpm": ">=7"
+  },
   "scripts": {
-    "clean": "yarn lerna run clean --parallel",
-    "build": "yarn lerna run build",
-    "test": "yarn lerna run test --parallel",
-    "test:coverage": "yarn lerna run test:coverage --parallel",
-    "lint": "yarn lerna run lint",
-    "lint:ts:check": "yarn lerna run lint:ts:check",
-    "lint:check": "yarn lerna run lint:check",
-    "lint:fix": "yarn lerna run lint:fix --parallel",
-    "lint:specs:fix": "yarn run markdownlint-cli2-fix \"./specs/**/*.md\"",
-    "lint:specs:check": "yarn run markdownlint-cli2  \"./specs/**/*.md\"",
-    "lint:specs:toc": "yarn run doctoc '--title=**Table of Contents**' ./specs",
+    "clean": "lerna run clean --parallel",
+    "build": "lerna run build",
+    "test": "lerna run test --parallel",
+    "test:coverage": "lerna run test:coverage --parallel",
+    "lint:ts:check": "lerna run lint:ts:check",
+    "lint:check": "lerna run lint:check",
+    "lint:fix": "lerna run lint:fix --parallel",
+    "lint:specs:fix": "markdownlint-cli2-fix \"./specs/**/*.md\"",
+    "lint:specs:check": "markdownlint-cli2  \"./specs/**/*.md\"",
+    "lint:specs:toc": "doctoc '--title=**Table of Contents**' ./specs",
```

Comment:
Could nuke all this for pnpm

`pnpm concurrently run clean --parallel`

I wouldn't bother though because we really should use nx.  There is a pr that is pretty much ready to merge now https://github.com/ethereum-optimism/optimism/pull/4778